config_cam_baseline_example.yaml

```diff
@@ -79,6 +79,15 @@ diag_basic_info:
     #Location where diagnostic plots are stored:
     cam_diag_plot_loc: /some/where/you/want/to/have/diag_plots #MUST EDIT!
 
+    #Use default variable plot settings?
+    #If "true", then variable-specific plotting attributes as defined in
+    #ADF/lib/adf_variable_defaults.yaml will be used:
+    use_defaults: false
```

Comment:
@brianpm can you change this value to `true` as requested by @andrewgettelman (unless you have a reason not to)?  Once that change has been made I'll go ahead and merge it.

Reply:
Yeah, just pushed this.